src/data/table/columninfo.cpp

```diff
@@ -96,17 +96,17 @@ ColumnInfo::ContiType ColumnInfo::getContiType() const
 }
 
 const ColumnInfo::ValueIndexes &
-ColumnInfo::discreteValueIndexes() const
+ColumnInfo::dimensionValueIndexes() const
 {
 	return valueIndexes;
 }
 
-const ColumnInfo::Values &ColumnInfo::discreteValues() const
+const ColumnInfo::Values &ColumnInfo::dimensionValues() const
```

Comment:
`dimensionValues` could be called `categories`. According to our nomenclature, Dimension series are containing categories, and Measure series are containing values.